cogs/displayinfo.py

```diff
@@ -33,85 +32,139 @@ class DisplayInfo(commands.Cog):
 
     def __init__(self,bot):
         self.bot = bot
-        self.timers = {}
-        self.current_collection = ""
-        self.current_guild_name = ""
+        self.current_collection = db["UserTracking"] # TODO: Move this secrets file
+        self.timers = self.current_collection.find_one() # Gets document from collection.
         self.current_date = str(date.today())
```

Comment:
FUTURE FIX (SEE TOP COMMENT): Possible issue with self.current_date since it isn't constantly being updated. Once we have the bot running overnight, we won't have this updated and it'll stay stuck on the day the bot started. We either need a timer to update this at 12 AM or we can just update it each time someone calls a function that requires usage of current_date.